scopes/component/isolator/isolator.main.runtime.ts

```diff
@@ -245,6 +246,9 @@ export class IsolatorMain {
     opts: IsolateComponentsOptions,
     legacyScope?: Scope
   ): Promise<CapsuleList> {
+    if (opts.installOptions) {
+      opts.installOptions.useNesting = this.dependencyResolver.hasRootComponents() && opts.installOptions.useNesting;
```

Comment:
you are going to override the original option, I think it's better to put it into a new local var